apps/data/elasticsearch/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:8.13.4
 
-HEALTHCHECK CMD curl -k https://elastic:$ELASTIC_PASSWORD@localhost:8311/_cluster/health
+HEALTHCHECK CMD curl -kf https://elastic:$ELASTIC_PASSWORD@localhost:8311/_cluster/health
```

Comment:
?

Reply:
Shouldn't really change functionality because of the way elasticsearch's healthcheck works, but the -f flag makes curl requests fail if the response isn't 200, otherwise 400/500 responses would be interpreted as a success. Decided to toss it in while I was updating the other health checks